app.vue

```diff
@@ -102,7 +102,7 @@ useProfileStore().$subscribe((_, state) => {
 });
 
 useHead({
-  titleTemplate: (chunk) => (chunk ? `BMM | ${chunk}` : "BMM"),
+  titleTemplate: (chunk) => (chunk ? `${chunk} - BMM` : "BMM"),
```

Comment:
I think at some point I suggested putting BMM first. But after looking at other websites (Figma, Google, Microsoft Azure, ...) I realized it's commonly done the other way.